infra/nightly-resources/chart.js

```diff
@@ -25,12 +25,28 @@ function getDataForBenchmarkRunMode(benchmark, runMode) {
   }
 }
 
+function getValue(entry) {
+  if (GLOBAL_DATA.chartMode === "absolute") {
+    return entry.mean;
+  } else if (GLOBAL_DATA.chartMode === "speedup") {
+    return entry.speedup;
```

Comment:
Could we compute the speedup here instead of storing it along side the data?
Rule of thumb is to touch the data as little as possible